docs/resources/gateway.md

```diff
@@ -182,6 +184,33 @@ resource "valtix_gateway" "gcp_hub_gw1" {
 
 For HUB mode INGRESS Gateway set the `security_type = INGRESS`
 
+### OCI Gateway (INGRESS Gateway in EDGE Mode)
+```hcl
+resource "valtix_gateway" "oci_gw1" {
+  name                      = "oci-gw1"
+  description               = "OCI Ingress Gateway 1"
+  csp_account_name          = valtix_cloud_account.oci_act.name
+  instance_type             = "OCI_E3_FLEX"
+  gateway_image             = var.gateway_image
+  gateway_state             = "ACTIVE"
+  mode                      = "EDGE"
+  security_type             = "INGRESS"
+	ssh_public_key            = file(var.ssh_public_key_file)
+  policy_rule_set_id        = valtix_policy_rule_set.ingress_policy_rule_set.id
+	region                    = "us-sanjose-1"

```

Comment:
The formatting needs to be fixed

Reply:
@kiranvaltix I've made the formatting changes.  Can you review again and confirm.